Fungi growth simulation/Assets/Direction.cs

```diff
@@ -113,4 +113,52 @@ public static Direction GetRandomDirection(Direction direction)
         List<Direction> neighbors = GetNeighbors(direction);
         return neighbors[Helper.Rnd.Next(neighbors.Count)];
     }
+
+
+    public static bool IsSquare(Vector3 vector) {
+        return vector.sqrMagnitude == 1;
+    }
+
+    public static Vector3[] GetBranchDirection(Direction direction) {
```

Comment:
What if we reused the GetNeighbors method, then shuffled the returned list and took 2 elements from it?
[shuffle](https://www.delftstack.com/howto/csharp/shuffle-a-list-in-csharp/)
[take](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.take?view=net-6.0)